runtime/runtime/src/lib.rs

```diff
@@ -2288,7 +2291,16 @@ impl<'a> ApplyProcessingState<'a> {
         incoming_receipts: &'a [Receipt],
         delayed_receipts: DelayedReceiptQueueWrapper,
     ) -> ApplyProcessingReceiptState<'a> {
+        let pipeline_manager = pipelining::ReceiptPreparationPipeline::new(
+            Arc::clone(&self.apply_state.config),
+            self.apply_state.cache.as_ref().map(|v| v.handle()),
+            self.epoch_info_provider.chain_id(),
+            self.apply_state.current_protocol_version,
+            todo!(),
+            // self.state_update.trie.storage, // TODO??
```

Comment:
Question to storage folks: what's the right way to access the underlying storage here? If there isn't one, what do you think would be the right way to expose it to this code?